c++/src/cpp_doc_config

```diff
@@ -38,7 +38,7 @@ PROJECT_NAME           =
 # could be handy for archiving the generated documentation or if some version
 # control system is used.
 
-PROJECT_NUMBER         = "1.14.4-1, currently under development"
+PROJECT_NUMBER         = "1.14.5-1, currently under development"
```

Comment:
Is 1.14.5 correct?

Reply:
According to the release process document, it is correct.

Reply:
Now that the release branch is created

Reply:
1.14.5 is the correct version for the hdf5_1_14 branch - for the next 1.14 release.  The version in the release branch hdf5_1_14_4 is 1.14.4-2 until the -2 is removed for the actual release.